signwriting_transcription/pose_to_signwriting/README.md

```diff
@@ -3,6 +3,18 @@
 We adapt the JoeyNMT framework to develop a Neural Machine Translation (NMT) model for
 transcribing human poses to SignWriting, a visual writing system for sign languages.
 
+## Usage
+```
+git clone https://github.com/sign-language-processing/signwriting-transcription.git
+cd signwriting-transcription
+pip install .[dev,pose_to_signwriting]
+cd ..
```

Comment:
no need for `cd ..` and no need for `[dev]` for inference (this is for development